orga/app/router.js

```diff
@@ -38,8 +38,8 @@ Router.map(function () {
       this.route('new', { path: '/creation' });
     });
     this.route('campaigns', { path: '/campagnes' }, function () {
-      this.route('list', { path: '/' }, function () {
-        this.route('my-campaigns');
+      this.route('list', { path: '/liste' }, function () {
```

Comment:
perso, je suis pas fan. Au début de Pix Orga, c'était ça, et c'est pas ouf parce que la route `/campaigns/{un truc}` c'est aussi la route pour le détail d'une campagne, qui déclenche un appel pour récupérer une campagne d'id `liste`, qui n'existe pas bien évidemment, et donc ça lance une erreur

Reply:
au début j'avais fait :
```js
this.route('campaigns', { path: '/campagnes' }, function () {
      this.route('list', { path: '/' }, function () {
        this.route('all-campaigns', { path: '/toutes' });
      });
      // ...
  });
```
Avec la route `campaigns.list` qui affichait "mes campagnes" mais ça posait des soucis pour la sélection des tabs (vu que `campaigns.list.all-campaigns` devenait un enfant de `campaigns.list`)
Si tu as une autre proposition je suis preneuse

Reply:
normalement tu peux faire
```suggestion
this.route('campaigns', { path: '/campagnes' }, function () { 
  this.route('my-campaigns', { path: '/' });
  this.route('all-campaigns', { path: '/toutes' });
});
```

Reply:
Done ✅